.gitignore

```diff
@@ -76,3 +76,4 @@ __pycache__
 
 # Allow to exist but don't include it in the repo
 user_song_list.h
+secrets.c
```

Comment:
Please revert. 

If you need it, add a .gitignore file in the folder with the file.
```suggestion
```